test/e2e/backup_test.go

```diff
@@ -148,10 +148,21 @@ func testEtcdBackupOperatorForS3Backup(t *testing.T) string {
 // testEtcdRestoreOperatorForS3Source tests if the restore-operator can restore an etcd cluster from an S3 restore source
 func testEtcdRestoreOperatorForS3Source(t *testing.T, s3Path string) {
 	f := framework.Global
+	suffix := fmt.Sprintf("-%d", rand.Uint64())
+	clusterName := "restore-tls-test" + suffix
+	memberPeerTLSSecret := "restore-etcd-peer-tls" + suffix
+	memberClientTLSSecret := "restore-etcd-server-tls" + suffix
+	operatorClientTLSSecret := "restore-etcd-client-tls" + suffix
+	err := e2eutil.PrepareTLS(clusterName, f.Namespace, memberPeerTLSSecret, memberClientTLSSecret, operatorClientTLSSecret)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
defer delete the secrets?
Also do the same thing for backup test?

Reply:
sure thing.